src/engine/types/mod.rs

```diff
@@ -353,3 +353,9 @@ impl<'a> From<&'a libudev::Device<'a>> for UdevEngineDevice {
         }
     }
 }
+
+#[derive(Debug, Clone, PartialEq)]
+pub struct DevicemapperInfo {
```

Comment:
I'm having trouble justifying why this summary information is a sufficient proxy, for two reasons:

1. What if the highest event_nr is one that hasn't changed? For example, for some reason the thinpool has the highest event number, but there's a filesystem that has experienced a change, and has had it's event number bumped. I think the pool should still ```event_on``` in that case.
2. ```device_count``` might be the same if a filesystem had been removed and another one added. But in that case, I think an ```event_on``` would still be indicated.

Maybe ```DevicemapperInfo``` can only be accurate if it's just a map of names to event numbers?

Reply:
Okay, this should be resolved!